application/template/NGSILD/goTemplate/main.go

```diff
@@ -0,0 +1,284 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	mux "github.com/gufranmirza/go-router"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"os/signal"
+	"strconv"
+	"syscall"
+)
+
+//var ctxUpdateBuffer []*ContextObject
+
+var isConfigured = false
+var brokerURL = ""
+var myReferenceURL = ""
+
+var inputEntityId = ""
+var inputEntityType = ""
+
+/*
+	read config.json
+*/
+func readConfig(fileName string) []interface{} {
+	config, e := ioutil.ReadFile(fileName)
+	if e != nil {
+		fmt.Printf("File Error: [%v]\n", e)
+		os.Exit(1)
+	}
+
+	dec := json.NewDecoder(bytes.NewReader(config))
+	var commands []interface{}
+	dec.Decode(&commands)
+
+	return commands
+}
+
+func startApp() {
+	fmt.Println("start to receive input data streams via a listening port")
+}
+
+func stopApp() {
+	fmt.Println("clean up the app")
```

Comment:
[minor] it should be stop and clean up the application

Reply:
Done